CMakeLists.txt

```diff
@@ -5,7 +5,9 @@ if(NOT CMAKE_BUILD_TYPE)
     message(STATUS "CMAKE_BUILD_TYPE not set, defaulting to MinSizeRel.")
 endif()
 
-set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}" "${CMAKE_SOURCE_DIR}/cmake")
+set(PDCURSES_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+set(PDCURSES_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR})

```

Comment:
Also, these two variables aren't necessary since CMake already automatically creates [`<PROJECT>_SOURCE_DIR`](https://cmake.org/cmake/help/latest/variable/PROJECT-NAME_SOURCE_DIR.html) and [`<PROJECT>_BINARY_DIR`](https://cmake.org/cmake/help/latest/variable/PROJECT-NAME_BINARY_DIR.html), in our case `pdcurses_SOURCE_DIR` and `pdcurses_BINARY_DIR` (notice the casing).